idaes/generic_models/properties/core/pure/tests/test_Perrys.py

```diff
@@ -227,3 +229,87 @@ def test_dens_mol_liq_comp_eqn_2(frame):
     assert value(expr) == pytest.approx(7.48194e10, rel=1e-4)
 
     assert_units_equivalent(expr, pyunits.mol/pyunits.m**3)
+
+
+@pytest.fixture()
```

Comment:
A minor observation, but I would only make a fixture if I intend to use the same code multiple times. In this case, this code is only used in one test, so I would normal just write it in the test directly.